src/main/java/de/dafuqs/spectrum/enchantments/CloversFavorEnchantment.java

```diff
@@ -37,10 +44,23 @@ public int getMaxPower(int level) {
 	public int getMaxLevel() {
 		return SpectrumCommon.CONFIG.CloversFavorMaxLevel;
 	}
+
+	@Override
+	public boolean isAcceptableItem(ItemStack stack) {
+		var item = stack.getItem();
+		if(SpectrumIntegrationPacks.isIntegrationPackActive(SpectrumIntegrationPacks.MALUM_ID) &&
+				item instanceof MalumScytheItem)
```

Comment:
Does this run in an instance without Malum installed?
Mentioning `MalumScytheItem` here may throw a class load error.

(same in later cases. But I'll not annotate every single one for now)